src/qibosoq/programs/base.py

```diff
@@ -244,23 +239,23 @@ def acquire(
         # if there are no readouts, temporaray set 1 so that qick can execute properly
         reads_per_rep = 1 if readouts_per_experiment == 0 else readouts_per_experiment
 
-        res = super().acquire(
+        res = self.acquire(  # pylint: disable=E1123
             soc,
             readouts_per_experiment=reads_per_rep,
-            load_pulses=load_pulses,
-            progress=progress,
-            debug=debug,
+            load_pulses=True,
```

Comment:
Just to understand myself: why are you hardcodingb these now?

It's a perfectly reasonable choice, and might save boilerplate, but I wanted to understand why now and why those values.

Reply:
They were already hardcoded in the rfsoc_server.py, I just moved them here. In any case, these are the default values, I am writing them explicitly for clarity and stability (if something changes in qick) but I could also remove them